data/defaults.yaml

```diff
@@ -54,7 +54,7 @@ prometheus::alertmanager::time_intervals: []
 prometheus::alertmanager::storage_path: '/var/lib/alertmanager'
 prometheus::alertmanager::templates: [ "%{lookup('prometheus::alertmanager::config_dir')}/*.tmpl" ]
 prometheus::alertmanager::user: 'alertmanager'
-prometheus::alertmanager::version: '0.21.0'
+prometheus::alertmanager::version: '0.27.0'
```

Comment:
if we update it, the PR title should reflect this as well.

Reply:
removed the version change, will update seperatly